src/InteractiveWindow/Editor/InteractiveWindow.cs

```diff
@@ -66,7 +62,6 @@ private enum ReplSpanKind
         private bool _adornmentToMinimize;
 
         private readonly IWpfTextView _textView;
-        private readonly History _history;
```

Comment:
Are there any fields in InteractiveWindow that can be used safely on multiple threads? If not, perhaps uiOnly should be an IInteractiveWindow implementation containing all state, and the outer instance simply delegates to the inner instance, after switching to the UI thread.